equinox/_module.py

```diff
@@ -477,10 +477,33 @@ def __init__(self, *args, **kwargs):
                     if not has_abstract_name:
                         # Invariant: abstract classes have names beginning with
                         # `Abstract`.
+                        inner = " "
+                        if len(cls.__abstractmethods__) > 0:
+                            inner = (
+                                inner
+                                + f"abstract methods: {list(cls.__abstractmethods__)}"
+                            )
+                        if len(cls.__abstractvars__) > 0:
+                            if len(inner) > 2:
+                                inner = inner + ", "
+                            inner = (
+                                inner
+                                + f"abstract variables: {list(cls.__abstractvars__)}"
+                            )
+                        if len(cls.__abstractclassvars__) > 0:
+                            if len(inner) > 2:
+                                inner = inner + ", "
+                            inner = (
+                                inner
+                                + f"abstract class variables: \
+                                {list(cls.__abstractclassvars__)}"
+                            )
+                        inner = inner + "."
                         raise TypeError(
                             "Abstract strict `eqx.Module`s must be named starting "
                             f"with 'Abstract' or '_Abstract'. Got {name} when defining "
-                            f"{cls.__module__}.{cls.__qualname__}.",
+                            f"{cls.__module__}.{cls.__qualname__}. For concrete "
+                            f"moduels, {name} has" + inner
```

Comment:
I don't think this will do the right for modules that are forced to be abstract? (`strict=StrictConfig(force_abstract=True)`)

Also typo `moduels`.

Reply:
Fixed typo and added a flag to ignore the inner if force is true.